README.md

```diff
@@ -1,24 +1,145 @@
 
-* System dependencies
 
-* Configuration
+# 📖 [hello-rails-react] <a name="about-project"></a>
 
-* Database creation
+hello rails react app is a web application that shows random greetings when a user refreshes the page.
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Express.js</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- Fetches random messages from the database
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](Coming Soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- gem install rails
+### Setup
+
+Clone this repository to your desired folder:
+
+- https://github.com/Paul-js-hub/hello-rails-react.git
+- cd hello-rails-react
+
+### Install
+
+Install this project with:
+
+- bundle install
+- npm install 
+- yarn install
+- rails db:create 
+- rails db:migrate
+- rails db:seed
+
+### Usage
+
+To run the project, execute the following command:
+
+- ./bin/dev
+
+### Run tests
+
+To run tests, run the following command:
+
+- bundle exec rspec
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="author"></a>
+
+👤 **Paul Onchera**
+
+- GitHub: [@paulonchera](https://github.com/Paul-js-hub)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/paul-onchera/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add a button that when clicked renders the random messages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Paul-js-hub/hello-rails-react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Hat tip to anyone whose code was used
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
**[Optional]**

- [ ] It's great that you have added a LICENSE.md file but the link to your LICENSE.md file is broken which is at the bottom of your readme file 🤔 Kindly fix this link to make your readme file descriptive 👍